Rules/CommonScripts/RegrowPlants.as

```diff
@@ -280,6 +286,14 @@ void onTick(CRules@ this)
 						server_CreateBlob(plants_stuff[plant], -1, tinfo.coords - Vec2f(0,tilesize));
 					}
 				}
+
+				random_grow = XORRandom(10000) * 0.0001f;
+
+				if (random_grow <= chicken_grow_chance && chicken_count < chicken_limit)
```

Comment:
FWIW this has the issue that if the chicken count is, say, `chicken_limit - 1`, dirt tiles that are early in the list (given how the list is laid out, this mean blue team tiles) will have a higher chance of spawning a chicken than the dirt tiles late in the list (red team ones) because if a chicken gets spawned early in the list, the chicken count limit will be hit.  
I don't think this makes much of a practical difference though, but if it actually ends up being anything noticeable this should be fixed.